docker-compose.yml

```
@@ -0,0 +1,16 @@
+version: "3.4"
+services:
+  postgres:
+    image: postgres:latest
+    container_name: fitflow-postgres
+    ports:
+      - "5432:5432"
+    expose:
+      - "5432"
+    volumes:
+      - ./postgres_data:/var/lib/postgresql/data
+      - ./docker/entrypoint-initdb.d:/docker-entrypoint-initdb.d
+    environment:
+      POSTGRES_USER: fitflow
+      POSTGRES_PASSWORD: fitflow
+      POSTGRES_DB: fitflow
```

Acho que aqui podemos seguir o padrão de definir o usuário como `root` e senha `password`

```suggestion
      POSTGRES_USER: root
      POSTGRES_PASSWORD: password
      POSTGRES_DB: fitflow
```